source/reference/operator/aggregation/rand.txt

```diff
@@ -0,0 +1,172 @@
+===================
+$rand (aggregation)
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. expression:: $rand
+
+   .. versionadded:: 4.4.2
+
+   The :expression:`$rand` operator returns a random float between 0
```

Comment:
[minor] All other aggregation operator pages lead with the verb, rather than using the operator name. Therefore, I suggest we remove "The :expression:`$rand` operator" from this sentence. And have this sentence read:

"Returns a random float between 0 and 1."

For a couple examples, notice the wording of the introductions of:

- https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCS-13883-add-rand-number-generator-v5.0/reference/operator/aggregation/sampleRate.html#exp._S_sampleRate

- https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCS-13883-add-rand-number-generator-v5.0/reference/operator/aggregation/setEquals.html

Reply:
Done